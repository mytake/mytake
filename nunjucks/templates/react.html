{% extends "base.html" %}

{% block endbody %}
  <!-- Dependencies -->
  <script src="https://unpkg.com/react@15/dist/react.js"></script>
  <script src="https://unpkg.com/react-dom@15/dist/react-dom.js"></script>

  <!-- Main -->
  <script src="./bundle-0.0.3.js"></script>
{% endblock %}
