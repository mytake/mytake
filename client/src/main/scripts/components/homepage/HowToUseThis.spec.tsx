import * as React from "react";
import * as renderer from "react-test-renderer";
import HowToUseThis from "./HowToUseThis";

jest.mock("./HomeSection", () => ({
  __esModule: true,
  default: "HomeSection",
}));

test("HowToUseThis", () => {
  const tree = renderer.create(<HowToUseThis />).toJSON();
  expect(tree).toMatchSnapshot();
});
