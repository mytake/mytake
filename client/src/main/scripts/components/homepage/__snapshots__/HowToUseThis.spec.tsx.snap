// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HowToUseThis 1`] = `
<HomeSection
  containerClassName="home__section--dark"
  innerContainerClassName="home__searchbar"
>
  <h1
    className="home__h1 home__h1--search"
  >
    The
     
    <a
      href="https://github.com/mytakedotorg/mtdo/blob/staging/DEV_QUICKSTART.md"
    >
      open source
    </a>
     
    political search engine
  </h1>
  <form
    action="javascript:void(0)"
    className="searchbar__form searchbar__form--home"
    onSubmit={[Function]}
  >
    <div
      className="searchbar__input-container searchbar__input-container--home"
    >
      <input
        className="searchbar__input searchbar__input--home"
        onChange={[Function]}
        placeholder="Search"
        results={5}
        type="search"
        value=""
      />
      <span
        className="searchbar__cancel searchbar__cancel--home searchbar__cancel--hidden"
        onClick={[Function]}
      >
        <svg
          fill="none"
          height={24}
          stroke="currentColor"
          strokeLinecap="round"
          strokeLinejoin="round"
          strokeWidth="2"
          viewBox="0 0 24 24"
          width={24}
          xmlns="http://www.w3.org/2000/svg"
        >
          <circle
            cx="12"
            cy="12"
            r="10"
          />
          <line
            x1="15"
            x2="9"
            y1="9"
            y2="15"
          />
          <line
            x1="9"
            x2="15"
            y1="9"
            y2="15"
          />
        </svg>
      </span>
    </div>
    <button
      className="searchbar__button searchbar__button--home"
      type="submit"
    >
      <span
        className="searchbar__text searchbar__text--home"
      >
        Search
      </span>
      <svg
        className="searchbar__icon searchbar__icon--home"
        fill="none"
        height={24}
        stroke="currentColor"
        strokeLinecap="round"
        strokeLinejoin="round"
        strokeWidth="2"
        viewBox="0 0 24 24"
        width={24}
        xmlns="http://www.w3.org/2000/svg"
      >
        <circle
          cx="11"
          cy="11"
          r="8"
        />
        <line
          x1="21"
          x2="16.65"
          y1="21"
          y2="16.65"
        />
      </svg>
    </button>
  </form>
  <ul
    className="home__ul"
  >
    <li
      className="home__li"
    >
      No editorializing, no filter, no algorithm
    </li>
    <li
      className="home__li"
    >
      Just the facts: who said what, and when
    </li>
    <li
      className="home__li"
    >
      Search every
       
      <a
        href="https://github.com/mytakedotorg/us-presidential-debates"
      >
        presidential debate in history
      </a>
      , more to come
    </li>
  </ul>
</HomeSection>
`;
